src/main/java/com/squareup/kotlinpoet/TypeSpec.kt

```diff
@@ -547,6 +556,9 @@ class TypeSpec private constructor(builder: TypeSpec.Builder) {
     @JvmStatic fun expectClassBuilder(name: String) = Builder(Kind.EXPECT_CLASS, name, null).apply {
       addModifiers(KModifier.EXPECT)
     }
+    @JvmStatic fun externalClassBuilder(name: String) = Builder(Kind.EXTERNAL_CLASS, name, null).apply {
```

Comment:
nit: looks like a newline before this declaration is missing

Reply:
Thanks, fixes